packages/mobile/src/components/audio-rewards/audioTier.ts

```diff
@@ -3,8 +3,9 @@ import BN from 'bn.js'
 
 export const WEI = new BN('1000000000000000000')
 
-export type BadgeTier = 'none' | 'bronze' | 'silver' | 'gold' | 'platinum'
-export const badgeTiers: { tier: BadgeTier; minAudio: BN }[] = [
+export type AudioTier = 'none' | 'bronze' | 'silver' | 'gold' | 'platinum'
+
+export const audioTierRequirements: { tier: AudioTier; minAudio: BN }[] = [
```

Comment:
same comment here re: upper snake casing constants, not sure if that's a thing in this codebase though

Reply:
yeah ill have a look around as well

Reply:
I also like to use `as const` with "constant" objects and arrays like this and `audioTierMap` above to prevent them from being accidentally modified by consumers. Not sure if that will mess up your types here though

Reply:
oh im looking at this: https://github.com/airbnb/javascript/#naming--uppercase so maybe this one we upper-case, but not the first one since it's not exported?

Reply:
ah! i like that, makes sense to me

Reply:
shall we add to Audius style guide? (i still have yet to add the stuff we talked about in product sync rip)

Reply:
yeah just added it!